integration/containerd/cri/integration-tests.sh

```diff
@@ -129,18 +130,35 @@ create_containerd_config() {
 	# Remove dots.  Dots are used by toml syntax as atribute separator
 	runtime="${runtime//./-}"
 
-	cat << EOT | sudo tee "${CONTAINERD_CONFIG_FILE}"
+	if [ $kata_annotations -eq 1 ]; then
```

Comment:
Does anything bad happen if we just always pass all annotations?

Reply:
> Does anything bad happen if we just always pass all annotations?

I think pass all annotations is OK for kata.